tests/default.rs

```diff
@@ -99,13 +99,50 @@ fn fuelup_default_nightly_and_nightly_date() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn fuelup_default_override_skip_install() -> Result<()> {
+    testcfg::setup(FuelupState::LatestWithBetaOverride, &|cfg| {
+        let output = cfg.fuelup_with_input(&["default"], b"N\n");
+        let triple = TargetTriple::from_host().unwrap();
+
+        assert!(output
+            .stdout
+            .starts_with(&format!("Using override toolchain 'beta-1-{triple}'")));
```

Comment:
nit: we're planning to remove support for betas 1-4 pretty soon, so it might be better to make these tests use beta-5